model-optimizer/mo/front_ng/extractor.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging as log
+import re
+from collections import defaultdict
+from copy import copy
+
+import numpy as np
+
+from mo.utils.error import Error
```

Comment:
Can we sort this imports according to PEP8 rules?

Reply:
Fixed